code/game/objects/effects/effect_system.dm

```diff
@@ -1198,17 +1198,17 @@ steam.start() -- spawns the effect
 
 
 //////////////////////////////////
-//////SPARKELS FIREWORKS
+//////SPARKLE FIREWORKS
 /////////////////////////////////
 ////////////////////////////
-/obj/effects/sparkels
-	name = "sparkel"

```

Comment:
Yes, all of those things. (It's an effect for one of the fireworks, if you're curious.)